### DISCUSSION TITLE
learner-long-life MNIST model

### DISCUSSION THREAD

**Discussion by learner-long-life** [TheEvergreenStateCollege/upper-division-cs#1398] [Show and tell]
`model-50epochs-10batch-0_2eta-80_18success.pbjson`

From code on branch `prototype-00`
* 50 epochs
* 10 sized batch
* 0.2 learning rate
* 80.18% success rate

Vanilla training in `network.py`, no optimizations yet.

Feel free to try out with `run_model.py` and let me know how it works for you.
Improvements and fine-tunings are welcome.

<hr /><em>This discussion was created from the release <a href='https://github.com/TheEvergreenStateCollege/upper-division-cs/releases/tag/'>learner-long-life MNIST model</a>.</em>
